mne_realtime/epochs.py

```diff
@@ -374,16 +375,14 @@ def _get_data(self, out=True, picks=None,  verbose=None):
         not on data loading.
         """
         if out:
-            epochs = list()
-            for epoch in self._epoch_queue:
-                epochs.append(epoch)
-
-            data = np.array(epochs)
-
-            if picks is not None:
-                picks = _picks_to_idx(self.info, picks)
-                data = data[:, picks]
-            return data
+            item = slice(None) if item is None else item
+            select = self._item_to_select(item)  # indices or slice
+            use_idx = np.arange(len(self._events))[select]
+            if picks is None:
+                picks = slice(None)
+            else:
+                picks = _picks_to_idx(self.info, picks, none='all', exclude=())
+            return np.array([self._epoch_queue[idx][picks] for idx in use_idx])
```

Comment:
will this return a single array or a array of arrays?

Reply:
It will return an ndarray with shape `(n_epochs, n_picks, n_times)`, which is what it was before I think